content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -695,6 +696,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `RotateKubeletServerCertificate`: Enable the rotation of the server TLS certificate on the kubelet.
   See [kubelet configuration](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#kubelet-configuration)
   for more details.
+- `RuntimeClassInImageCriApi` : Enables images to be pulled based on runtime class.
```

Comment:
```suggestion
- `RuntimeClassInImageCriApi` : Enables images to be pulled based on
  the [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/) of the Pods
  that reference them.
```
Container images don't have a runtime class; Pods do.

Reply:
Didn't mean to say container images had runtime class :) The feature gate name is indicating runtime class in **image CRI** which we have already introduced with this KEP.

Reply:
I will add the suggested text to the description